cmds/docs/index.js

```diff
@@ -49,6 +50,9 @@ exports.run = function (opts) {
   }
 
   const files = fs.readdirSync(folder).filter(file => file.endsWith('.md') || file.endsWith('.markdown'));
+  if (files.length === 0) {
+    return Promise.reject(new Error(`We were unable to locate Markdown files in ${folder}.`));
```

Comment:
Adding this error message because if a user tries to use `rdme docs` with a directory that has subdirectories containing Markdown, we currently can't handle those Markdown files as `rdme docs` can't handle category creation and management right now.

https://app.asana.com/0/1178132843162889/1183431246035890/f